### PR TITLE
[IOTDB-5485] Rename StorageGroup to Database in ConfigNode

### PR DESCRIPTION
**Pull request by CRZbulabula**
Just rename by refactor

### CODE REVIEW

**Comment by OneSizeFitsQuorum** [comment]
Please fix these code smell~ BTW, you can use IDEA to generate this methods.
image

**Review by OneSizeFitsQuorum — APPROVED** [review]
LGTM

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -887,7 +887,7 @@ Import-CSV supports specify data type and no need quotation for text value
 * IOTDB-947 Fix error when counting node with wildcard
 * IOTDB-949 Align by device doesn't support 'T*' in path.
 * IOTDB-956 Filter will be missed in group by time align by device
-* IOTDB-963 Redo deleteStorageGroupPlan failed when recovering
+* IOTDB-963 Redo deleteDatabasePlan failed when recovering
```
Comment (SpriCoder) [inline RELEASE_NOTES.md:890]: No need to update release notes?

**Review by SpriCoder — APPROVED** [review]
LGTM

File: RELEASE_NOTES.md (hunk 1 above)
Comment (CRZbulabula) [reply RELEASE_NOTES.md]: Fixed~

**Pull request merged**